release/models/network-instance/openconfig-route-summary.yang

```diff
@@ -0,0 +1,96 @@
+module openconfig-route-summary {
+  yang-version "1";
+
+  namespace "http://openconfig.net/yang/route-summary";
+
+  prefix "oc-rtsummary";
+
+  import openconfig-extensions { prefix "oc-ext"; }
+  import openconfig-yang-types { prefix "oc-yang-types"; }
+  import openconfig-policy-types { prefix "oc-pol-types"; }
+  import openconfig-network-instance { prefix "oc-ni"; }
+
+  organization
+    "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+    www.openconfig.net";
+
+  description
+    "This module provides summary of route counts per route type for each network
+    instance.";
+
+  oc-ext:openconfig-version "1.0.0";
+
+  revision "2022-12-21" {
+    description
+      "Initial version.";
+    reference "1.0.0";
+  }
+
+  grouping route-counter {
+    leaf route-type {
```

Comment:
Given that this is a reference to the protocol that installed this, should we just call this `protocol` or `source`?

Reply:
I think `protocol` is more descriptive than `source`.

If we are changing the key to be called `protocol`, should we change the grouping name from `route-counter` to `protocol-counter` as well?

Reply:
I'm OK with it being `route-counter` just broken down by protocol.

Reply:
We decided to rename it to `source` because some routes may not come from protocols.